### ISSUE TITLE
Changes to backgound script [background.js] doesn't trigger hot reloading

### ISSUE THREAD

**Issue by psych0der** [jhen0409/react-chrome-extension-boilerplate#52]
Unlike other files, changes made to background.js doesn't propagate automatically and requires manual reloading of the chrome extension. Any workaround?

**Comment by jhen0409** [comment]
Yes it doesn't, if you want background scripts to support Hot reload, you have to make more extra works, like [glenjamin/ultimate-hot-reloading-example](https://github.com/glenjamin/ultimate-hot-reloading-example) made Express server hot reload.

Also, a correctly behavior is automatically full-reload (see [webpack/replace/log-apply-result.js#L16-L18](https://github.com/jhen0409/react-chrome-extension-boilerplate/blob/master/webpack/replace/log-apply-result.js#L16-L18)), because we change to `webpack-hot-middleware`, so currently it doesn't work. The code should move to `webpack-hot-middleware/process-update.js`, I'll update it.
